webapp/channels/src/store/index.js

```diff
@@ -22,9 +22,21 @@ window.Observable = Observable;
 
 const localForage = extendPrototype(baseLocalForage);
 
-export default function configureStore(preloadedState) {
+export default function configureStore(preloadedState, testPluginReducerKeys) {
```

Comment:
Instead of passing an array of keys into here, I'd like to keep the explicit test logic out of the store setup, so could you change this to take an object containing additional reducers and make `configureOrMockStore` construct the dummy reducers to pass into here? That way, the new parameter here is basically this functions' version of `configureServiceStore`'s `appReducers` parameter